applications/FreeSurfaceApplication/python_scripts/free_surface_analysis.py

```diff
@@ -0,0 +1,25 @@
+# Kratos imports
+import KratosMultiphysics
+from KratosMultiphysics.FreeSurfaceApplication.edgebased_levelset_solver import EdgeBasedLevelSetSolver
+from KratosMultiphysics.analysis_stage import AnalysisStage
+
+
+class FreeSurfaceAnalysis(AnalysisStage):
+
+    def __init__(self,
+                 model: KratosMultiphysics.Model,
+                 parameters: KratosMultiphysics.Parameters):
+        # Base class constructor defines:
+        # - self.model
+        # - self.project_parameters
+        # - self.echo_level
+        # - self.parallel_type
+        AnalysisStage.__init__(self, model, parameters)
+
+
+    def _CreateSolver(self) -> EdgeBasedLevelSetSolver:
+        return EdgeBasedLevelSetSolver(self.model, self.project_parameters["solver_settings"].Clone())
```

Comment:
```suggestion
        return EdgeBasedLevelSetSolver(self.model, self.project_parameters["solver_settings"])
```

this should not be done
the idea is to be able to see the full and validated params